rust/common-runtime/src/sandbox/wasmtime/compile/bindings.rs

```diff
@@ -114,7 +77,13 @@ where
             references: ResourceTable::default(),
 
             view_resources: ResourceTable::default(),
-            view_ctx: WasiCtx::builder().build(),
+            view_ctx: WasiCtx::builder()
+                .allow_tcp(false)
+                .allow_udp(false)
+                .allow_ip_name_lookup(false)
+                .allow_blocking_current_thread(false)
+                .inherit_stdout()
```

Comment:
What's the purpose of inheriting stdout? stdlib's stdout requires synchronization as well as a point of contention
https://github.com/bytecodealliance/wasmtime/blob/158a8f59e7b1bc095465ee8d197356fb031c5fa3/crates/wasi/src/stdio.rs#L294

Reply:
It's only because this is the only way I know to see logs from inside the Wasm component without additional effort. I will file an issue to create a purpose-built `common:logger` WIT and move to using that.